plugins/outputs/opentelemetry/opentelemetry.go

```diff
@@ -23,11 +23,12 @@ import (
 
 	"github.com/influxdata/telegraf"
 	"github.com/influxdata/telegraf/config"
+	"github.com/influxdata/telegraf/internal"
 	"github.com/influxdata/telegraf/plugins/common/tls"
 	"github.com/influxdata/telegraf/plugins/outputs"
 )
 
-var userAgent = fmt.Sprintf("telegraf (%s/%s)", runtime.GOOS, runtime.GOARCH)
+var userAgent = fmt.Sprintf("telegraf %s (%s/%s)", internal.Version(), runtime.GOOS, runtime.GOARCH)
```

Comment:
All plugins should be using the same user agent. The project has standardized on internal.ProductToken() as the user agent.

https://github.com/influxdata/telegraf/blob/64c85eb4f83c7710cfe62cede20cbd7ff873f2b1/internal/internal.go#L57

I see that the opentelemetry plugin wasn't using it before, but if we're changing the plugin, we should be switching to the standard, not adding a different user agent format.